src/components/UserManagedList.js

```diff
@@ -168,6 +173,21 @@ export default class UserManagedList extends SystemManagedList {
     return { coordinates }
   }
 
+  getListFromFossaPackage(dependencies) {
+    const coordinates = []
+    for (const dependency in dependencies) {
+      const locator = dependencies[dependency].locator.split(/[\s+$/]+/)
+      coordinates.push({
+        type: locator[0],
+        provider: 'npmjs',
```

Comment:
Should other package managers be supported besides NPM?

Reply:
Honestly, i'm not so experienced about Fossa , so i don't know which kind of package managers could be supported.
If you take a look to the code, we are actually forcing the same thing for the `package-lock` file, so i guess is an expected behavior?

Reply:
@geneh We could actually improve this code, and match the field `type` with the related provider (ex. `npm -> npmjs`) . 
We can extend this improvement also for the `package-lock` file parsing.

Thoughts?

Reply:
Good! Isn't package-lock relevant only to NPM? Fossa seems to support multiple package managers: https://github.com/fossas/fossa-cli/blob/master/README.md#Supported-Environments